packages/core/src/indexing/postgres/__tests__/postgres-index-api.fixture.ts

```diff
@@ -0,0 +1,805 @@
+/**
+ * Stream content test data-set
+ */
+export const STREAM_TEST_DATA_PROFILE = JSON.stringify([
```

Comment:
why such a large and complex test content?  I feel like it makes it seem like we're using all this for something.  If a simple dictionary with a 2-3 fields would test just as well, let's just use that and keep it simple.

Also, I don't think MIDs support arrays at the top level, the content hast to be a dictionary object